sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -54,25 +52,24 @@ def __init__(
         ----------
         scope : App
         construct_id : str
-        sds_id : str
-            Name suffix for stack
         opensearch: OpenSearch
             This class depends on opensearch, which is built with opensearch_stack.py
         dynamodb_stack: DynamoDb
             This class depends on dynamodb_stack, which is built with
             opensearch_stack.py
         processing_step_function_arn:
             This has step function arn
-        env : Environment
-            Account and region
         """
         super().__init__(scope, construct_id, env=env, **kwargs)
+        # Get the current account number so we can use it in the bucket names
+        account = env.account
```

Comment:
Is there some way to put an account alias in this env variable that gets passed around all over the place?

Reply:
This is an instance of the AWS Environment class, so I'm not sure we can add other names to it. We could just pass in the account_config here as well... @tech3371 said she was going to rearrange all of this, so I'd prefer leaving this alone for now.